doc/modules/reference.rst

```diff
@@ -40,7 +40,8 @@ uses.
    :toctree: generated/
    :template: function.rst
 
-   sym_to_vec
```

Comment:
I would suggest to put it back.

The reason being that, this function deprecated but not removed yet. So, if user is still using it (in their pipelines for example) he would understand that this function is renamed and will be removed.

I would stress more on keeping warning message clear and mentioning release name too **0.4**.